packages/jbrowse-plugin-apollo/src/components/DeleteAssembly.tsx

```diff
@@ -46,7 +46,17 @@ export function DeleteAssembly({
   const [selectedInternetAcount, setSelectedInternetAcount] = useState(
     apolloInternetAccounts[0],
   )
-  const assemblies = useAssemblies(internetAccounts, setErrorMessage)
+
+  const assemblies: AssemblyData[] = useAssemblies(
+    internetAccounts,
+    setErrorMessage,
+  )
+
+  useEffect(() => {
+    if (assemblies.length > 0) {
```

Comment:
It would be good here to also check if there already is a selected assembly, since we don't need to set one in that case.